node/src/accountant/mod.rs

```diff
@@ -1742,129 +1688,105 @@ pub mod tests {
+        let blockchain_bridge = blockchain_bridge.retrieve_transactions_response(Ok(vec![]));
+        let earning_wallet = make_wallet("someearningwallet");
+        let payable_dao = PayableDaoMock::new().non_pending_payables_result(vec![]);
+        let receivable_dao = ReceivableDaoMock::new()
+            .new_delinquencies_result(vec![])
+            .paid_delinquencies_result(vec![]);
+        let system = System::new("accountant_logs_if_no_transactions_were_detected");
+        let config_mock = PersistentConfigurationMock::new().start_block_result(Ok(5));
         let config = bc_from_ac_plus_earning_wallet(
             AccountantConfig {
-                payable_scan_interval: Duration::from_secs(10_000),
-                payment_received_scan_interval: Duration::from_millis(100),
+                payables_scan_interval: Duration::from_secs(10_000),
+                receivables_scan_interval: Duration::from_millis(100),
             },
             earning_wallet.clone(),
         );
+        let subject = make_subject(
+            Some(config),
+            Some(payable_dao),
+            Some(receivable_dao),
+            None,
+            Some(config_mock),
+        );
+        let accountant_addr = subject.start();
+        let accountant_subs = Accountant::make_subs_from(&accountant_addr);
+        let peer_actors = peer_actors_builder()
+            .blockchain_bridge(blockchain_bridge)
+            .build();
+        send_bind_message!(accountant_subs, peer_actors);
+        send_start_message!(accountant_subs);
 
-        thread::spawn(move || {
-            let system = System::new("accountant_logs_if_no_transactions_were_detected");
-            let payable_dao = PayableDaoMock::new().non_pending_payables_result(vec![]);
-            let receivable_dao = ReceivableDaoMock::new()
-                .new_delinquencies_result(vec![])
-                .paid_delinquencies_result(vec![]);
-            let config_mock = PersistentConfigurationMock::new().start_block_result(Ok(5));
-            let subject = make_subject(
-                Some(config),
-                Some(payable_dao),
-                Some(receivable_dao),
-                None,
-                Some(config_mock),
-            );
-            let peer_actors = peer_actors_builder()
-                .blockchain_bridge(blockchain_bridge)
-                .accountant(accountant_mock)
-                .build();
-            let subject_addr: Addr<Accountant> = subject.start();
-            let subject_subs = Accountant::make_subs_from(&subject_addr);
-
-            send_bind_message!(subject_subs, peer_actors);
-            send_start_message!(subject_subs);
-
-            system.run();
-        });
+        System::current().stop();
+        system.run();
 
-        blockchain_bridge_awaiter.await_message_count(1);
+        let blockchain_bridge_recorder = blockchain_bridge_recording_arc.lock().unwrap();
+        assert_eq!(blockchain_bridge_recorder.len(), 1);
```

Comment:
If you record out just a single message you don't need to use the big apparatus below to get the message. You simply write something like  
let received_msg = blockchain_bridge_recorder.get_record::<RetrieveTransactions>(0);